src/modules/app/StateType.js

```diff
@@ -21,6 +23,10 @@ export type StateType = {
   categories: Payload<CategoriesMapModel>,
   extras: Payload<Array<ExtraModel>>,
   events: Payload<Array<EventModel>>,
+  localNews: Payload<Array<LocalNewsModel>>,
+  localNewsElement: Payload<LocalNewsModel>,
```

Comment:
I think it make sense here, since we have separate endpoint for each of them, by this we have control on the response type, for example if the endpoint for localNews return different fields from what localNewsElement endpoint return.
Does that make sense?

Reply:
Since we have separate endpoints I think it make sense to be like this. What are you suggesting here?

Reply:
Yeah I also think so. Would be too much overhead changing it to one element. So it's fine.